### ISSUE TITLE
HTTP request in loop (every 1 second) with valid response

### ISSUE THREAD

**Issue by Neubivljiv** [ufna/VaRest#258]
Hello,

I have added HTTP requests in Game Instance. If I call request via Player State or Game State it will send HTTP requests in loop (every second) with valid JSON response. 
However, if I call request from player controller (or character or probably user widget) it will work fine. 

Not sure this is a bug or something else.

**Comment by ufna** [comment]
Please provide an example.

**Comment by Neubivljiv** [comment]
Ok. This is HTTP request in Game Instance
![GameInstance](https://user-images.githubusercontent.com/339178/73122566-d4686380-3f86-11ea-9ba6-42afc14fbdec.jpg)

When I call this from Game State (get game instance -> cast to My_GameInstance ) it will return valid JSON response but will repeat every second. Also tried from Player State. 

This is JSON response (and it repeats every 1 sec):

> JSON(
> {"data":"ok,0"}
> )JSON

I am using UE 4.24.1

**Comment by Neubivljiv** [comment]
I did some more tests. 
Basically, I want to server send HTTP requests for clients in the game. So I called this event from Game Mode (event for HTTP req is in game instance) and I put IsDedicatedServer check, cast it to game state (to get all players) and call HTTP event in the game instance. 
I received 2 responses:
server: (empty response)
client: valid JSON response
and it repeats. 
Seems that empty response is causing loop. 
I could not get players data from Game Mode, but from Game State and when I call it from game state HTTP request is sent on client side.
